packages/blocks/src/page-block/edgeless/components/toolbar/brush/brush-tool-button.ts

```diff
@@ -51,6 +51,15 @@ export class EdgelessBrushToolButton extends LitElement {
     :host {
       display: flex;
     }
+    edgeless-tool-icon-button.icon-container:hover {
+      background: var(--affine-background-overlay-panel-color);
+    }
+    edgeless-tool-icon-button svg {
+      transform: translateY(3px);
+    }
```

Comment:
Tweaking SVG position in CSS is not the best practice.

Reply:
Is there any way to achieve stretch the pen and eraser when hover on it only with css? If it is, maybe I don't need to add eventListener and manipulate the svg manually...